cfme/fixtures/ansible_fixtures.py

```diff
@@ -156,3 +164,81 @@ def order_ansible_service_in_ops_ui(appliance, ansible_catalog_item,
     service = MyService(appliance, cat_item_name)
     if service.exists:
         service.delete()
+
+
+@pytest.fixture(scope="module")
+def ansible_catalog_item_create_empty_file(appliance, ansible_repository):
+    collection = appliance.collections.catalog_items
+    cat_item = collection.create(
+        collection.ANSIBLE_PLAYBOOK,
+        fauxfactory.gen_alphanumeric(15, start="create_file_"),
```

Comment:
Note, this only leaves 3 characters for randomization.

Reply:
I think enough for `nCr` as cat item name :+1: